cmd/root.go

```diff
@@ -66,6 +67,12 @@ var rootCmd = &cobra.Command{
 			handleError(err)
 		} else if cui {
 			query = ""
+		} else if presetQuery != "" {
+			if val, ok := tui.Queries[presetQuery]; ok {
+				query = val
+			} else {
+				fmt.Printf("Incorrect index Queries are from 1 - %d", len(tui.Queries))
```

Comment:
let's construct a new error here and use the `handleError`?
```go
handleError(errors.New("..."))
```

and instead of mentioning indexes, let's report "unknown preset query: %s" or something along those lines